src/lightkurve/io/detect.py

```diff
@@ -47,6 +48,10 @@ def detect_filetype(hdulist: HDUList) -> str:
     if "mit/qlp" in hdulist[0].header.get("origin", "").lower():
         return "QLP"
 
+    # Is it a GSFC-ELEANOR-LITE light curve?
+    if hdulist[0].header.get("LITE") == True:
```

Comment:
Will this work if this keyword doesn't exist? Maybe @barentsen can weigh in on whether this is the right way to do things, or if this is not good practice?

Reply:
Is the reader intended to work with GSFC-ELEANOR-LITE exclusively, or any FITS file generated from vanilla `eleanor`?

Based on what I've scanned the codes in the vanilla `eleanor`, it appears that the following headers are generated by the custom version of GSFC-ELEANOR-LITE:
```
AUTHOR = 'Adina D. Feinstein'
TVERSION= '0.0.1.dev0'
GITHUB = 'https://github.com/afeinstein20/eleanor'
```

Reply:
Yes, it is exclusively intended to work with GSFC-ELEANOR-LITE.

Reply:
> Will this work if this keyword doesn't exist? Maybe @barentsen can weigh in on whether this is the right way to do things, or if this is not good practice?

If there is no keyword, detect_filetype returns “None”

Reply:
> Is the reader intended to work with GSFC-ELEANOR-LITE exclusively, or any FITS file generated from vanilla `eleanor`?
> 
> Based on what I've scanned the codes in the vanilla `eleanor`, it appears that the following headers are generated by the custom version of GSFC-ELEANOR-LITE:
> 
> ```
> AUTHOR = 'Adina D. Feinstein'
> TVERSION= '0.0.1.dev0'
> GITHUB = 'https://github.com/afeinstein20/eleanor'
> ```

Yes, it is exclusively intended to work with GSFC-ELEANOR-LITE.

Reply:
If the reader is to work with GSFC-ELEANOR-LITE exclusively, detection via `LITE = T` would not work, because vanilla `eleanor` also produces `LITE=T` header if the users chooses to do so in  [`TargetData.save()`](https://adina.feinste.in/eleanor/api.html#eleanor.TargetData.save)

Reply:
Good point. @christinahedges, @barentsen can you weight it on this? Should we use "AUTHOR" and/or "TVERSION" to distinguish between eleanor and elenor-lite?

Reply:
Hang on, I'm just catching up on all this, but is there any reason not to let this generically read in any eleanor light curves? Right now lk can't handle any eleanor light curves. Why not allow any, whether user-generated or from GSFC? As far as I know, none of the differences between lite and regular will have any effect on the rest of the changes.